mongodbatlas/clusters_test.go

```diff
@@ -36,7 +36,9 @@ func TestClusters_ListClusters(t *testing.T) {
 						},
 						"awsPrivateLinkSrv": {
 							"vpce-0d00c26273372c6ef": "mongodb+srv://cluster0-pl-0-auylw.mongodb.net"
-						}
+						},
+						"private": "mongodb://cluster0-shard-00-00-auylw.mongodb.net:27017,cluster0-shard-00-01-auylw.mongodb.net:27017,cluster0-shard-00-02-auylw.mongodb.net:27017/?ssl=true&authSource=admin&replicaSet=Cluster0-shard-0",
+						"privateSrv": "mongodb+srv://cluster0-auylw.mongodb.net"
```

Comment:
```suggestion
						"private": "mongodb://cluster0-shard-00-00-pri.auylw.mongodb.net:27017,cluster0-shard-00-01-pri.auylw.mongodb.net:27017,cluster0-shard-00-02-pri.auylw.mongodb.net:27017/?ssl=true&authSource=admin&replicaSet=Cluster0-shard-0",
						"privateSrv": "mongodb+srv://cluster0-pri.auylw.mongodb.net"
```
to keep it accurate. We should use this for all the other tests too.

Reply:
ok thank you, I'll change it, I copy pasted from standard because I couldn't find an example for this case in mongodb docs

Reply:
Yep, I realized that it wasn't in the docs either.